cluster-provision/k8s/1.18/manifests/cni.do-not-change.yaml

```diff
@@ -798,3 +792,5 @@ metadata:
 
 ---
 # Source: calico/templates/configure-canal.yaml
+
```

Comment:
Please drop these trailing lines unless they come from the downloaded source

Reply:
its came from the download source, all the changes there, the file is identical now